pkg/quota/v1/evaluator/core/services.go

```diff
@@ -127,14 +127,29 @@ func (p *serviceEvaluator) Usage(item runtime.Object) (corev1.ResourceList, erro
 		value := resource.NewQuantity(int64(ports), resource.DecimalSI)
 		result[corev1.ResourceServicesNodePorts] = *value
 	case corev1.ServiceTypeLoadBalancer:
-		// load balancer services need to count node ports and load balancers
-		value := resource.NewQuantity(int64(ports), resource.DecimalSI)
-		result[corev1.ResourceServicesNodePorts] = *value
+		// load balancer services need to count node ports. If creation of node ports
+		// is suspressed only ports with explicit NodePort values are counted.
```

Comment:
We should add to this - nodeports won't be allocated yet, so we can't simply count the actual values.  We need to look at the intent.